packages/template-retail-react-app/app/commerce-api/index.test.js

```diff
@@ -27,6 +27,13 @@ import {
 
 jest.mock('cross-fetch', () => jest.requireActual('jest-fetch-mock'))
 
+jest.mock('./utils', () => {
+    const originalModule = jest.requireActual('./utils')
+    return {
+        ...originalModule
+    }
+})
```

Comment:
this code block does nothing 😛 ... it mocks the module by the module itself.
```suggestion
```

Reply:
I added this because tests in this file will fail if it relies on the `isTokenValid` mock in the global `jest-setup` file but I will use the other approach to generate a valid JWT instead!